src/style.js

```diff
@@ -225,3 +225,10 @@ export function applyInlineStyles(selection, style) {
     }
   }
 }
+
+export function applyFrameAnchor({frameAnchor}, {width, height, marginTop, marginRight, marginBottom, marginLeft}) {
+  return [
+    /-?left$/.test(frameAnchor) ? marginLeft : /-?right$/.test(frameAnchor) ? width - marginRight : (marginLeft + width - marginRight) / 2,
+    /^top-?/.test(frameAnchor) ? marginTop : /^bottom-?/.test(frameAnchor) ? height - marginBottom : (marginTop + height - marginBottom) / 2
```

Comment:
I don't think we need the `-?` here since the keywords have already been validated.

Reply:
We do, because you can have e.g. “top” or “top-left” and we want to match on both.

Reply:
Oh, I see what you mean, because “top” will match both, and nothing else. That’s true, but I was a little worried that we’d introduce another name in the future that would coincidentally match the same prefix. I think it’s clearer to have the hyphen as an explicit delimiter even though it’s not strictly necessary.

Reply:
I meant that matching "top" or "left" seems to be enough. Just a nit.

Reply:
Okay, I thought about `/^top($|-)/` instead, but we’d have to change this code if we introduce new frameAnchor values anyway, so it feels fine to take your suggestion.